utt/utt-client-api/src/User.cpp

```diff
@@ -583,7 +586,8 @@ void User::debugOutput() const {
     std::cout << '\n';
   };
 
-  for (const auto& coin : pImpl_->coins_) {
+  for (const auto& [_, coin] : pImpl_->coins_) {
+    (void)_;
```

Comment:
how did it compile before without a warning?

Reply:
previously, coins was a vector